rfd/0178-app-custom-icons.md

```diff
@@ -0,0 +1,97 @@
+---
+author: Lisa Kim (lisa@goteleport.com)
+state: draft
+---
+
+# RFD 178 - Application Resource Custom Icon
+
+## Required Approvers
+
+- Engineering: @romant && @avatus && (someone from the scale team))
+
+## What
+
+Support custom icon uploads for resources through yaml. We can start with feature request [#4712](https://github.com/gravitational/teleport/issues/4712) where
+users provide a URL and Teleport's auth server will download the image and store it in the backend.
```

Comment:
Hmm, this seems a little scary from a SSRF perspective, no? Allowing users the ability to make Teleport connect to any arbitrary URL could divulge information about the network or be used as a vector to introduce malware.